src/Core/ContainerInterface.php

```diff
@@ -40,18 +41,17 @@ interface ContainerInterface extends PsrInterface
      * @param mixed $concrete Either the classname an interface or name resolves to.
      *   Can also be a constructed object, Closure, or null. When null, the `$id` parameter will
      *   be used as the concrete class name.
-     * @param bool $shared Set to true to make a service shared.
      * @return \League\Container\Definition\DefinitionInterface
      */
-    public function add(string $id, $concrete = null, bool $shared = false): DefinitionInterface;
+    public function add(string $id, $concrete = null): DefinitionInterface;
```

Comment:
League's container now has an explicit `addShared()` method, should we add the same to our interface?

Reply:
We should.